### PR TITLE
Contact section PR

### PR DESCRIPTION
**Pull request by laurianerzb**
### Contact section pull request

#Added contact section in index.html
#Added styles for contact section in styles.css
#Checked for linters error

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -250,3 +246,94 @@ nav {
...
+  background-size: cover;
+  background-color: #6070ff;
+  border-top-left-radius: 98px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-left: 10px;
+  padding: 10%;
+}
```
Comment (Santiago220991) [inline styles.css:264]: - Kindly make sure that your project renders correctly in screen sizes from 375px to desktop screen sizes. :eyes: :eyes: 
image

File: styles.css [hunk 2]
```diff
@@ -250,3 +246,94 @@ nav {
...
+  color: #6070ff;
+  border: none;
+  padding: 0.8rem 2rem;
+  align-self: flex-start;
+}
+
+.c-form button:hover {
+  color: red;
+  background-color: #fff;
+}
```
Comment (Santiago220991) [inline styles.css:339]: - Kindly make sure you are applying button interactions such as hover, active (when it's pressed), disabled, and enable (default state) styling as shown in the Figma template. :+1: :smiley: :smiley: 
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @laurianerzb , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -197,5 +197,16 @@ <h2>About<br />Myself</h2>
...
+    <!-----------------CONTACT SECTION---------------------------->
+    <section class="contact-section">
+          <h2 class="contact-title">Contact me</h2>
+          <p class="contact-desc">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+        <form class="c-form-container" action="https://formspree.io/f/xyyvwogq" method="post">
+              <input type="text" id="name" name="username" maxlength="30" placeholder="name here" required/>
+              <input type="email" id="email" name="email" placeholder="email here" required/>
+              <textarea id="message" name="user_message" maxlength="500" placeholder="write your message here" required></textarea>
+              <button class="c-button" type="submit">Get in touch</button>
+        </form>
```
Comment (mckent05) [inline index.html:209]: - Just like the previous reviewer suggested, please ensure that your design matches the given Figma design you have chosen 🙏 . Please note that replicating the exact Figma template gives confidence to recruiters in your ability to replicate exact business requirements. Please try to fix this.

Your design currently looks like this:
contact project

Figma design -
figma contact

File: styles.css [hunk 4]
```diff
@@ -250,3 +246,97 @@ nav {
...
+  border: none;
+  padding: 0.8rem 2rem;
+  align-self: flex-start;
+  border-radius: 8px;
+}
+
+.c-form-container button:hover {
+  color: red;
+  font-size: 19px;
+}
```
Comment (mckent05) [inline styles.css:342]: **OPTIONAL** - Please ensure that you implement button interaction such as hover, enabled etc..  Please note that replicating the exact Figma template gives confidence to recruiters in your ability to replicate exact business requirements. Please try to fix this.

Figma template (please pay attention to the border, color, bg color) -
figma 1button

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi guys,

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost there


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 5]
```diff
@@ -250,3 +248,97 @@ nav {
...
+  align-self: flex-start;
+  border-radius: 8px;
+  margin: 10px 40px;
+  height: 48px;
+}
+
+.c-form-container button:hover {
+  color: red;
+  font-size: 19px;
+}
```
Comment (Alfredbis29) [inline styles.css:344]: - [ ]  Very good design so far, you did a great job to make changes here :+1:  , but we can improve something to make it attractive to recruiters.
Here you can see how the width of the page is fixed but it should have a `width:100%` to re-flow according to the screen width :wink:  , Kindly fix this, you can see that your project is not fitting all the screen  :point_down: 
contactformrereview

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @laurianerzb, 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- Good readMe :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

 ## Required Changes ♻️

- [ ] You performed an outstanding job on this assignment. :+1:  However, I believe it would be good ideal if you included `the line bar` at the bottom of your `footer`, like we do on the Figma design. This will improve the appearance of your portfolio. :hugs: :wink: 

Figma Design 
linebar

_Check the comments under the review._

## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @laurianerzb , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**